Your order

Chillbroz opened the pull request:
Привіт, відправляю на узгодження

Chillbroz committed:
start

Chillbroz committed:
mobile

Chillbroz committed:
Merge branch 'main' of https://github.com/stelmakh1991/DreamTeamGoIT-project-21

Chillbroz committed:
monday

Chillbroz committed:
Merge branch 'main' of https://github.com/stelmakh1991/DreamTeamGoIT-project-21

Chillbroz committed:
Merge branch 'your_order'

Chillbroz committed:
mobile styles

Chillbroz committed:
tablet

Chillbroz committed:
desc

Chillbroz committed:
pre-final

stelmakh1991 commented on src/partials/your_order.html:
```diff
@@ -0,0 +1,43 @@
+                <form class="order-form">
+                    <div class="order-item">
+                        <label class="order-label" for="user-name"></label>
+                        <input type="text" class="form-input" name="user-name" id="user-name" placeholder="Enter your full name" />
```
<input type="text" class="form-input" name="user-name" id="user-name" placeholder="Enter your full name" required />

stelmakh1991 commented on src/partials/your_order.html:
```diff
@@ -0,0 +1,43 @@
+                    </div>
+                    <div class="order-item">
+                        <label class="order-label" for="user-email"></label>
+                        <input type="email" class="form-input" name="user-email" id="user-email" placeholder="Enter your email" />
```
<input type="email" class="form-input" name="user-email" id="user-email" placeholder="Enter your email" required />

stelmakh1991 commented on src/partials/your_order.html:
```diff
@@ -0,0 +1,43 @@
+            </div>
+            <div class="ornament">
+                <picture>
+                    <source srcset="../img/your_order/desctop1440/image1-desc1440.jpg 1x, ../img/your_order/desctop1440/image1-desc1440-2x.jpg 2x" media="(min-width: 1440px)">
```
<source srcset="../img/your_order/desctop1440/image1-desc1440.jpg 1x, ../img/your_order/desctop1440/image1-desc1440-2x.jpg 2x" media="(min-width: 1440px)" />

stelmakh1991 commented on src/partials/your_order.html:
```diff
@@ -0,0 +1,43 @@
+            <div class="ornament">
+                <picture>
+                    <source srcset="../img/your_order/desctop1440/image1-desc1440.jpg 1x, ../img/your_order/desctop1440/image1-desc1440-2x.jpg 2x" media="(min-width: 1440px)">
+                    <source srcset="../img/your_order/desctop/image1-desc.jpg 1x, ../img/your_order/desctop/image1-desc-2x.jpg 2x" media="(min-width: 1280px)">
```
<source srcset="../img/your_order/desctop/image1-desc.jpg 1x, ../img/your_order/desctop/image1-desc-2x.jpg 2x" media="(min-width: 1280px)" />

stelmakh1991 commented on src/partials/your_order.html:
```diff
@@ -0,0 +1,43 @@
+                <picture>
+                    <source srcset="../img/your_order/desctop1440/image1-desc1440.jpg 1x, ../img/your_order/desctop1440/image1-desc1440-2x.jpg 2x" media="(min-width: 1440px)">
+                    <source srcset="../img/your_order/desctop/image1-desc.jpg 1x, ../img/your_order/desctop/image1-desc-2x.jpg 2x" media="(min-width: 1280px)">
+                    <source srcset="../img/your_order/tablet/Image1-tab.jpg 1x, ../img/your_order/tablet/Image1-tab-2x.jpg 2x" media="(min-width: 768px)">
```
<source srcset="../img/your_order/tablet/Image1-tab.jpg 1x, ../img/your_order/tablet/Image1-tab-2x.jpg 2x" media="(min-width: 768px)" />

stelmakh1991 commented on src/css/your_order.css:
```diff
@@ -0,0 +1,265 @@
+}
+
+.form-input {
+    min-width: 100%;
```
width: 100%;

stelmakh1991 commented on src/css/your_order.css:
```diff
@@ -0,0 +1,265 @@
+}
+
+.form-comm {
+    min-width: 100%;
```
width: 100%;

Chillbroz committed:
pre-final w/o header

stelmakh1991 reviewed (changes requested):
поправ кілька моментів

Chillbroz committed:
pre-final correction

Виправив min-width на width у стилях.
Огорнув текст кнопки у спан, бо валідатор ругався на параграф. до инпутів додав pattern та required

Chillbroz commented:
Виправив min-width на width у стилях.
Огорнув текст кнопки у спан, бо валідатор ругався на параграф. до инпутів додав pattern та required

stelmakh1991 committed:
Merge branch 'main' into your_order

stelmakh1991 committed:
Merge branch 'main' into your_order
